cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -64,6 +64,12 @@ public final class Controllers {
     public static final String COUNT = "count";
     public static final String TIME = "time";
     public static final String RESULTS = "results";
+    public static final String NEW_BASIN_ID = "newBasinId";
```

Comment:
Remove any of these that aren't being used as query params.

Reply:
Fixed in latest push